run_action.py

```diff
@@ -541,24 +539,23 @@ def dismiss_change_requests(
         # Avoid triggering abuse detection
         time.sleep(10)
 
-    if auto_resolve_conversations:
-        resolve_conversations(
-            github_token=github_token,
-            repo=repo,
-            pull_request_id=pull_request_id,
-            github_api_timeout=github_api_timeout,
-            single_comment_markers=single_comment_markers,
-        )
-
 
+# pylint: disable=too-many-locals, too-many-arguments, too-many-positional-arguments
 def conversation_threads_to_close(
-    repo, pr_number, github_token, github_api_timeout, single_comment_markers
+    repo,
+    pr_number,
+    github_token,
+    github_api_timeout,
+    single_comment_markers,
+    comment_paths=None,
 ):
     """Generator of unresolved conversation threads to close
 
     Uses the GitHub GraphQL API to get conversation threads for the given PR.
     Then filters for unresolved threads and those that have been created by the action.
     """
+    if comment_paths is None:
+        comment_paths = set()
```

Comment:
I suppose we can just set the `comment_paths` argument by default to an empty set instead of `None` and remove this check?